test/unit/EchoClientTest.php

```diff
@@ -97,6 +97,88 @@ function testRecentEvents()
         $this->assertEquals(array($expectedEvent),$result);
     }
 
+    function testGetEvents()
+    {
+        $this->setRequiredDefines();
+
+        $stubPersonaClient = $this->getMock('\Talis\Persona\Client\Tokens', array(), array(), '', false);
```

Comment:
There's functionally no difference here, but using `$this->getMockBuilder()/getMock()` seems like it would be a little cleaner, since you're mostly using defaults.